docs/digger/admin/managing-digger-logs-artifacts.asciidoc

```diff
@@ -0,0 +1,31 @@
+== Managing Digger Build Logs and Artifacts
+App builds started by Digger produce build logs and build artifacts.
+Over time, these will build up. There are a number of options to manage the buildup. If you are using Digger with RHMAP, these options can be specified
+by setting environment variables in the `millicore` deployment(???) configuration in
```

Comment:
it might be enough to list the environment variables that are required and point the user to the [OpenShift documentation](https://docs.openshift.com/container-platform/3.4/dev_guide/environment_variables.html#set-environment-variables) on how to update env variables.